src/renderer/components/Editor/EditMarker.tsx

```diff
@@ -0,0 +1,32 @@
+import { Transcription, Word } from 'sharedTypes';
+import EditMarkerSvg from '../../assets/EditMarkerSvg';
+
+interface Props {
+  transcription: Transcription;
```

Comment:
I believe passing this as a prop means that if the transcription changes at all, every single edit marker will re-render.

Perhaps instead pass `prevWord: Word | null` and `nextWord: Word | null` - that way, they will only re-render when relevant changes happen.